### DISCUSSION TITLE
2012/05/21/%E8%B1%86%E8%B1%86%E6%80%95%E9%A3%9E%E6%9C%BA

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#355] [Announcements]
# 2012/05/21/%E8%B1%86%E8%B1%86%E6%80%95%E9%A3%9E%E6%9C%BA

昨天做了件蠢事，害得豆豆被吓到了，自己也半宿没睡。豆豆平时看见天上的飞机就兴奋的啊啊直叫，我想他这么喜欢飞机，要是去机场看见大飞机一定更高兴。于是昨天就带他到了虹桥机场外围，选了个可以看见跑道的开阔位置让他过过瘾。当巨大的飞机轰鸣着从头顶掠过时，豆豆并没有预想的那么兴奋，后来才意识到他可能有点害怕吧。白天还好，等晚上睡觉，豆豆就开始噩梦不断，快三点的时候更是哇哇大哭起来，要我们把灯打开才敢再睡觉了。

https://blog.qizhen.xyz/2012/05/21/%E8%B1%86%E8%B1%86%E6%80%95%E9%A3%9E%E6%9C%BA

**Comment by ruanqizhen** [comment]
之前博客下的留言：

wuximing
2013年05月11日 10:42 编辑
这个对于不同小朋友，感受都是不同的，有个体差异的。凡是第一次经历的，只有家长多观察一下咯。

回复
